README.md

```diff
@@ -73,7 +74,7 @@ Plug 'SirVer/ultisnips' | Plug 'honza/vim-snippets'
 Plug 'scrooloose/nerdtree', { 'on':  'NERDTreeToggle' }
 Plug 'tpope/vim-fireplace', { 'for': 'clojure' }
 
-" Using git URL
+" Any valid git URL is allowed
 Plug 'https://github.com/junegunn/vim-github-dashboard.git'
```

Comment:
@vheon Ya I meant you.I think junegunn's drawing a blank like me. Ah well, I guess we'll just use stick to the full url github one that is present.